### PR TITLE
bump version of alpine in test images

### PR DESCRIPTION
**Pull request by mikedanese**
To pick up fix for:

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-5021
https://talosintelligence.com/vulnerability_reports/TALOS-2019-0782

I haven't pushed these images yet so expect tests to fail.

/kind cleanup
/sig testing

```release-note
NONE
```

### CODE REVIEW

File: test/images/sample-apiserver/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.10
+1.11
```
Comment (mkumatag) [inline test/images/sample-apiserver/VERSION]: as the code is based on `1.10` release, lets bump this version some something meaningful like `1.10v1`

File: test/utils/image/manifest.go [hunk 2]
```diff
@@ -193,25 +193,25 @@ const (
...
 func initImageConfigs() map[int]Config {
 	configs := map[int]Config{}
-	configs[CRDConversionWebhook] = Config{e2eRegistry, "crd-conversion-webhook", "1.13rev2"}
-	configs[AdmissionWebhook] = Config{e2eRegistry, "webhook", "1.14v1"}
+	configs[CRDConversionWebhook] = Config{e2eRegistry, "crd-conversion-webhook", "1.1r"}
+	configs[AdmissionWebhook] = Config{e2eRegistry, "webhook", "1.15"}
 	configs[Agnhost] = Config{e2eRegistry, "agnhost", "1.0"}
-	configs[APIServer] = Config{e2eRegistry, "sample-apiserver", "1.10"}
-	configs[AppArmorLoader] = Config{e2eRegistry, "apparmor-loader", "1.0"}
+	configs[APIServer] = Config{e2eRegistry, "sample-apiserver", "1.11"}
```
Comment (mkumatag) [inline test/utils/image/manifest.go]: same here as well

File: test/images/crd-conversion-webhook/VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.13rev2
+1.14
```
Comment (mkumatag) [inline test/images/crd-conversion-webhook/VERSION]: this can be `1.15` as we are already in `1.15` release

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
In few places, we can just migrate to distroless static image. Will track that in a different issue.

meanwhile can someone @ixdy @listx can build and push the image after addressing the review comments to pass the tests?

/hold

File: test/images/sample-apiserver/VERSION (hunk 1 above)
Comment (mikedanese) [reply test/images/sample-apiserver/VERSION]: But when we build these now, they will be built against 1.15... In general the versioning system doesn't make sense in this directory.

**Comment by mikedanese** [comment]
@mkumatag I was thinking about migrating to distroless static but right now I just want to make the scanners happy. I'll look into it and do the easy ones. I have permission and am able to build and push these images, so I don't need assistance.

File: test/images/sample-apiserver/VERSION (hunk 1 above)
Comment (mkumatag) [reply test/images/sample-apiserver/VERSION]: na! this image is little different, I see `1.10` code is checked out in the dockerfile here - https://github.com/kubernetes/kubernetes/blob/master/test/images/sample-apiserver/Dockerfile#L26

**Comment by mkumatag** [comment]
> @mkumatag I was thinking about migrating to distroless static but right now I just want to make the scanners happy. I'll look into it and do the easy ones. I have permission and am able to build and push these images, so I don't need assistance.

Make sense, created https://github.com/kubernetes/kubernetes/issues/77635 to track this work.

**Comment by mkumatag** [comment]
> I have permission and am able to build and push these images, so I don't need assistance.

we can push these images to pass the tests after fixing the image version 👍

**Comment by mikedanese** [comment]
@mkumatag PTAL. I converted the images that don't have deps on the base image to distorless static.

**Comment by mkumatag** [comment]
> @mkumatag PTAL. I converted the images that don't have deps on the base image to distorless static.

thanks @mikedanese change looks okay to me, can you rebuild the images and push them?

/lgtm

**Comment by mikedanese** [comment]
@mkumatag will do. Thanks for the review.

**Comment by mikedanese** [comment]
pushed images with:
```
for image in apparmor-loader crd-conversion-webhook dnsutils hostexec ipc-utils net no-snat-test-proxy no-snat-test nonewprivs redis sample-apiserver sample-device-plugin webhook; do
  make all-push QEMUVERSION=v3.1.0-3 WHAT=$image;
done;
```

**Comment by mikedanese** [comment]
@mkumatag had to adjust a few things to get tests passing again. PTAL

File: test/images/redis-slave/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+FROM BASEIMAGE
+
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+
+RUN apk add --no-cache redis
+
+COPY redis.conf /etc/redis.conf
+
+EXPOSE 6379
+VOLUME /data
```
Comment (tallclair) [inline test/images/redis-slave/Dockerfile]: How about just basing this on the redis image? Everything above here is the same. Or better yet, we could get rid of this image, and just dump run.sh into the pod spec.

File: test/images/redis/redis.conf [hunk 5]
```diff
@@ -1,2 +1,3 @@
 bind 0.0.0.0
 dir /data
+protected-mode no
```
Comment (tallclair) [inline test/images/redis/redis.conf]: What does this do, and why is it included in this PR?

File: test/images/sample-device-plugin/Dockerfile [hunk 6]
```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
+FROM gcr.io/distroless/static:latest
+
 ADD sampledeviceplugin /sampledeviceplugin
+
 ENTRYPOINT ["/sampledeviceplugin", "-alsologtostderr"]
```
Comment (tallclair) [inline test/images/sample-device-plugin/Dockerfile]: nit
```suggestion
 ENTRYPOINT ["/sampledeviceplugin", "-logtostderr"]
```

**Review by tallclair — COMMENTED** [review]
/cc @yuwenma

File: cluster/addons/fluentd-elasticsearch/es-statefulset.yaml [hunk 7]
```diff
@@ -98,7 +98,7 @@ spec:
       # If your OS already sets up this number to a higher value, feel free
       # to remove this init container.
       initContainers:
-      - image: alpine:3.6
+      - image: alpine:3.9
```
Comment (oomichi) [inline cluster/addons/fluentd-elasticsearch/es-statefulset.yaml]: After creating this PR, alpine 3.10 has been released.
Can we update to the version at this time?

**Review by oomichi — COMMENTED** [review]
This seems necessary to be rebased, and just one comment

**Pull request closed without merging**